### PR TITLE
Cursors!

### PR DESCRIPTION
**Pull request by ggdaltoso**
Why not exporting all of them?

image

*_Not all of them implemented_

thanks again, @belakm ✨

### CODE REVIEW

File: packages/core/components/GlobalStyle/GlobalStyle.tsx [hunk 1]
```diff
@@ -1,5 +1,6 @@
-import { createGlobalStyle } from '@xstyled/styled-components';
```
Comment (mmartinsoliveira) [inline packages/core/components/GlobalStyle/GlobalStyle.tsx:1]: what do you think to use the createGlobalStyle from xstyled?

File: packages/core/components/GlobalStyle/GlobalStyle.tsx [hunk 2]
```diff
@@ -66,7 +29,7 @@ const GlobalStyle = createGlobalStyle`
     src: url('${videowoff2}') format('woff2'),
          url('${videowoff}') format('woff'),
          url('${videottf}') format('truetype'),
-         url('${videoeot}?#iefix') format('embedded-opentype');
+         url('${videoeot}.eot?#iefix') format('embedded-opentype');
```
Comment (mmartinsoliveira) [inline packages/core/components/GlobalStyle/GlobalStyle.tsx:32]: what do you think to remove this change?

File: packages/core/components/GlobalStyle/Scrollbar.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+  ::-webkit-scrollbar-button {
+    box-sizing: border-box;
+    display: inline-block;
+    background: ${({ theme }) => theme.colors.material};
+    color: ${({ theme }) => theme.colors.materialText};
+    ${scrollBarBorder()}
+    display: block;
+    outline-offset: -2px;
+    height: 17px;
+    width: 17px;
```
Comment (mmartinsoliveira) [inline packages/core/components/GlobalStyle/Scrollbar.tsx:73]:
```suggestion
    height: 17;
    width: 17;
```

File: packages/core/components/GlobalStyle/Scrollbar.tsx [hunk 4]
```diff
@@ -0,0 +1,110 @@
...
+  ].join(',')};
+  background-color: ${secondaryColor};
+  background-size: ${`${pixelSize * 2}px ${pixelSize * 2}px`};
+  background-position: 0 0, ${`${pixelSize}px ${pixelSize}px`};
+`;
+
+export const scrollbars = css`
+  ::-webkit-scrollbar {
+    width: 17px;
+    height: 17px;
```
Comment (mmartinsoliveira) [inline packages/core/components/GlobalStyle/Scrollbar.tsx:44]:
```suggestion
    width: 17;
    height: 17;
```

File: packages/core/components/GlobalStyle/Scrollbar.tsx [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    outline-offset: -2px;
+    height: 17px;
+    width: 17px;
+    background-repeat: no-repeat;
+    background-size: 80%;
+    background-position: 0 0;
+  }
+  ::-webkit-scrollbar-button:active,
+  ::-webkit-scrollbar-button:active {
+    background-position: 0 1px;
```
Comment (mmartinsoliveira) [inline packages/core/components/GlobalStyle/Scrollbar.tsx:80]:
```suggestion
    background-position: 0 1;
```

**Review by mmartinsoliveira — CHANGES_REQUESTED** [review]

File: packages/core/components/GlobalStyle/Scrollbar.tsx (hunk 5 above)
Comment (ggdaltoso) [reply packages/core/components/GlobalStyle/Scrollbar.tsx:80]: Done!

File: packages/core/components/GlobalStyle/Scrollbar.tsx (hunk 4 above)
Comment (ggdaltoso) [reply packages/core/components/GlobalStyle/Scrollbar.tsx:44]: Done!

File: packages/core/components/GlobalStyle/Scrollbar.tsx (hunk 3 above)
Comment (ggdaltoso) [reply packages/core/components/GlobalStyle/Scrollbar.tsx:73]: Done!

File: packages/core/components/GlobalStyle/GlobalStyle.tsx (hunk 2 above)
Comment (ggdaltoso) [reply packages/core/components/GlobalStyle/GlobalStyle.tsx:32]: I couldn't agree more!

File: packages/core/components/GlobalStyle/GlobalStyle.tsx (hunk 1 above)
Comment (ggdaltoso) [reply packages/core/components/GlobalStyle/GlobalStyle.tsx:1]: Changed!

**Pull request merged**